tests/garage/tf/q_functions/test_continuous_cnn_q_function.py

```diff
@@ -0,0 +1,316 @@
+import pickle
+from unittest import mock
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+from garage.tf.envs import TfEnv
+from garage.tf.q_functions import ContinuousCNNQFunction
+from tests.fixtures import TfGraphTestCase
+from tests.fixtures.envs.dummy import DummyDiscretePixelEnv
+from tests.fixtures.models import SimpleCNNModel
+from tests.fixtures.models import SimpleCNNModelWithMaxPooling
+from tests.fixtures.models import SimpleMLPMergeModel
+
+
+class TestContinuousCNNQFunction(TfGraphTestCase):
+
+    # yapf: disable
+    @pytest.mark.parametrize('filter_dims, num_filters, strides', [
+        ((3, ), (5, ), (1, )),
+        ((3, ), (5, ), (2, )),
+        ((3, 3), (5, 5), (1, 1)),
+    ])
+    # yapf: enable
+    def test_get_qval(self, filter_dims, num_filters, strides):
+        env = TfEnv(DummyDiscretePixelEnv())
+        obs = env.reset()
+
+        with mock.patch(('garage.tf.models.'
+                         'cnn_mlp_merge_model.CNNModel'),
+                        new=SimpleCNNModel):
+            with mock.patch(('garage.tf.models.'
+                             'cnn_mlp_merge_model.MLPMergeModel'),
+                            new=SimpleMLPMergeModel):
+                qf = ContinuousCNNQFunction(env_spec=env.spec,
+                                            filter_dims=filter_dims,
+                                            num_filters=num_filters,
+                                            strides=strides)
+
+        action_dim = env.action_space.shape
+
+        obs, _, _, _ = env.step(1)
+
+        act = np.full(action_dim, 0.5)
+        expected_output = np.full((1, ), 0.5)
+
+        outputs = qf.get_qval([obs], [act])
+
+        assert np.array_equal(outputs[0], expected_output)
+
+        outputs = qf.get_qval([obs, obs, obs], [act, act, act])
+
+        for output in outputs:
+            assert np.array_equal(output, expected_output)
+
+    # yapf: disable
```

Comment:
can you please disable the specific yapf warning that you are encountering

Reply:
this just disables auto-formatting.